<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SOURISH-PC" failures="1" tests="1" name="cucumber.TestNGTestRunner" time="20.132" errors="0" timestamp="2025-06-28T14:56:12 IST" skipped="0">
  <testcase classname="cucumber.TestNGTestRunner" name="runScenario" time="20.132">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at sourishBakshiFrameworkAuto.StepDefinitions.StepDefinitionImpl.message_displayed_conf_page(StepDefinitionImpl.java:68)
at âœ½."Thankyou for the order" message is displayed on Confirmation page(file:///C:/Users/SOURISH/cucumber/SeleniumFrameworkDesignSb/src/test/java/cucumber/PurchaseOrder.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- cucumber.TestNGTestRunner -->
